spec/views/miq_ae_class/_ns_list.html.haml_spec.rb

```diff
@@ -8,76 +8,6 @@ def setup_namespace_form(dom)
       assign(:edit, :new => {:ns_name => dom.name, :ns_description => "test"})
     end
 
-    describe "Git domain" do
-      before do
-        dom = FactoryBot.create(:miq_ae_git_domain)
-        setup_namespace_form(dom)
-      end
-
-      it "with name_readonly and description_readonly set to true", :js => true do
-        render
-        expect(response).to have_css("namespace-form[name_readonly='true']")
-        expect(response).to have_css("namespace-form[description_readonly='true']")
-      end
-
-      it "with ae_ns_domain set to true", :js => true do
-        render
-        expect(response).to have_css("namespace-form[ae_ns_domain='true']")
-      end
-
-      it "with empty namespace_path", :js => true do
-        render
-        expect(response).to have_css("namespace-form[namespace_path='']")
-      end
-    end
-
-    describe "User domain" do
-      before do
-        dom = FactoryBot.create(:miq_ae_domain)
-        setup_namespace_form(dom)
-      end
-
-      it "with name_readonly and description_readonly set to false", :js => true do
-        render
-        expect(response).to have_css("namespace-form[name_readonly='false']")
-        expect(response).to have_css("namespace-form[description_readonly='false']")
-      end
-
-      it "with ae_ns_domain set to true", :js => true do
-        render
-        expect(response).to have_css("namespace-form[ae_ns_domain='true']")
-      end
-
-      it "with empty namespace_path", :js => true do
-        render
-        expect(response).to have_css("namespace-form[namespace_path='']")
-      end
-    end
-
-    describe "Namespace" do
-      before do
-        dom = FactoryBot.create(:miq_ae_namespace, :parent => FactoryBot.create(:miq_ae_domain))
-        assign(:sb, :namespace_path => 'namespace/path')
-        setup_namespace_form(dom)
-      end
-
-      it "with name_readonly and description_readonly set to false", :js => true do
-        render
-        expect(response).to have_css("namespace-form[name_readonly='false']")
-        expect(response).to have_css("namespace-form[description_readonly='false']")
-      end
-
-      it "with ae_ns_domain set to true", :js => true do
-        render
-        expect(response).to have_css("namespace-form[ae_ns_domain='false']")
-      end
-
-      it "Check Git enabled domain", :js => true do
-        render
-        expect(response).to have_css("namespace-form[namespace_path='namespace/path']")
-      end
-    end
-
```

Comment:
I'm a little concerned about removing these specs, but I guess we no longer need them after https://github.com/ManageIQ/manageiq-api/pull/647 is in.